internal/gh/project_next.go

```diff
@@ -0,0 +1,103 @@
+package gh
+
+import (
+	"context"
+
+	"github.com/shurcooL/githubv4"
+)
+
+// GraphQLFields represent a list of GitHub PNFs (Project Next Field).
+type GraphQLFields struct {
+	TotalCount githubv4.Int
+	Nodes      []GraphQLField
+}
+
+// GraphQLField represents a single GitHub PNF (Project Next Field).
+type GraphQLField struct {
+	ID       githubv4.ID
+	Name     githubv4.String
+	DataType githubv4.String
+	Settings githubv4.String
+}
+
+// GraphQLFieldValues represents list of GitHub PNFVs (Project Next Field Value).
+type GraphQLFieldValues struct {
+	TotalCount githubv4.Int
+	Nodes      []GraphQLFieldValue
+}
+
+// GraphQLFieldValue represents a single GitHub PNFV (Project Next Field Value).
+type GraphQLFieldValue struct {
+	ID           githubv4.ID
+	Value        githubv4.String
+	ProjectField GraphQLField
+
+	// ValueTitle is a special field to display the value in a more human-readable way.
+	ValueTitle string `graphql:"value"`
+}
+
+// GraphQLItems represents a list of GitHub PNIs (Project Next Item).
+type GraphQLItems struct {
+	TotalCount githubv4.Int
+	Nodes      []GraphQLItem
+}
+
+// GraphQLItem represents a single GitHub PNI (Project Next Item).
+type GraphQLItem struct {
+	ID          githubv4.ID
+	FieldValues GraphQLFieldValues `graphql:"fieldValues(first: $fieldsMax)"`
+}
+
+// Querier describes a GitHub GraphQL client that can make a query.
+type Querier interface {
+	Query(ctx context.Context, q interface{}, vars map[string]interface{}) error
```

Comment:
Let's document that method. Also, `any`, not `interface{}`

Reply:
fixed